server/deadfish.cpp

```diff
@@ -0,0 +1,76 @@
+#include "deadfish.hpp"
+#include "../common/constants.hpp"
+
+// HidingSpot
+
+HidingSpot::HidingSpot(const FlatBuffGenerated::HidingSpot* fb_Hs) {
+    this->name = fb_Hs->name()->str();
+    b2BodyDef myBodyDef;
+	myBodyDef.type = b2_staticBody;
+	myBodyDef.position.Set(fb_Hs->pos()->x(), fb_Hs->pos()->y());
+	body = gameState.b2world->CreateBody(&myBodyDef);
+
+	b2FixtureDef fixtureDef;
+    fixtureDef.isSensor = true; // makes hiding spot detect collision but allow movement
+    if (fb_Hs->ellipse()) {
```

Comment:
If the ellipse variable makes the distinction between the fixture being a circle and a polygon maybe change it to "circle"? Or add an enum with the type of the shape

Reply:
It proved to be redundant, and thus was removed. `ellipse` is no more.